modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheUtils.java

```diff
@@ -1888,6 +1890,21 @@ private static DataStorageConfiguration extractDataStorage(ClusterNode node, Ign
         return null;
     }
 
+    /**
+     * @return {@code true} if persistence is enabled for a default data region, {@code false} if not.
+     */
+    public static boolean isDefaultDataRegionPersistence(DataStorageConfiguration cfg) {
+        if (cfg == null)
+            return false;
+
+        DataRegionConfiguration defaultRegionCfg = cfg.getDefaultDataRegionConfiguration();
```

Comment:
abbreviation plugin suggests to change name to 'dfltRegionCfg'

Reply:
Fixed